runtime/queries/apex/locals.scm

```diff
@@ -1,41 +1,51 @@
 ; declarations
-
 (class_declaration) @local.scope
+
 (method_declaration) @local.scope
+
 (constructor_declaration) @local.scope
+
 (enum_declaration) @local.scope
+
 (enhanced_for_statement) @local.scope
 
 ; if/else
-
 (if_statement) @local.scope
+
 (if_statement
-  consequence: (_) @local.scope) ; if body in case there are no braces
```

Comment:
I think we should leave inline comment alone, or at least move them _above_ the line